### DISCUSSION TITLE
Is there any plans to support geo location based queries?

### DISCUSSION THREAD

**Discussion by rgururaj** [eclipse-ditto/ditto#2035] [General]
I may have a bunch of robots that are moving continuously on a floor that is sending lat, lng which will be persisted on the robot.
Now if I want to find all the robots near a geo coordinate live or from the history (timeseries) find within a timewindow of timeseries within a given radius of geo coordinate.

**Comment by thjaeckle** [comment]
I don't have any plans personally - if you need that, I suggest contributing to the project 👍
